ui/components/multichain/asset-picker-amount/asset-picker-modal/index.scss

```diff
@@ -37,10 +37,10 @@
       }
     }
 
-    &--disabled {
-      @include disabled-token {
-        opacity: 0.4;
-      }
+    &--disabled,
+    &:disabled {
+      opacity: var(--opacity-disabled);
+      cursor: not-allowed;
```

Comment:
Not seeing the correct cursor here

<img width="419" alt="Screenshot 2024-07-01 at 4 29 53 PM" src="https://github.com/MetaMask/metamask-extension/assets/44588480/1e9f7560-3b9a-4b9c-81e6-204555fed0bc">

Reply:
done